### PR TITLE
add instructions for installing plugins

### PR DESCRIPTION
**Pull request by Tr4pSt3R**
Updates README with installation instructions from NeoBundle to Dein

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,6 +51,31 @@ Dein.vim is a dark powered Vim/Neovim plugin manager.
     :call dein#install()
     ```
 
+## Installing Plugins
+Append these instructions to the end of your `~/.vimrc` file.
```
Comment (Shougo) [inline README.md]: `vimrc` should be `vimrc`/`init.vim` file.

File: README.md [hunk 2]
```diff
@@ -51,6 +51,31 @@ Dein.vim is a dark powered Vim/Neovim plugin manager.
...
+Append these instructions to the end of your `~/.vimrc` file. 
+
+1. Open `~/.vimrc`
+
+2. Add vim plugins without dependencies to your `~/.vimrc` file.
+   ```vim
+   call dein#add('mileszs/ack.vim')
+   ```
+
+3. Add vim plugins with dependencies this line to your `~/.vimrc file. Repeat for as many plugins as you want to install.
```
Comment (Shougo) [inline README.md]: The backquote is needed.

File: README.md [hunk 3]
```diff
@@ -51,6 +51,31 @@ Dein.vim is a dark powered Vim/Neovim plugin manager.
...
+   ```vim 
+   call dein#add('mattn/gist-vim', {'depends': 'mattn/webapi-vim'})
+   ```
+
+4. Reload Vim
+   ```bash
+   :so ~/.vimrc
```
Comment (Shougo) [inline README.md]: `:so $MYVIMRC`

File: README.md (hunk 1 above)
Comment (Tr4pSt3R) [inline README.md]: thanks @Shougo I will change it accordingly

File: README.md (hunk 2 above)
Comment (Tr4pSt3R) [inline README.md]: I shall put it in, thanks @Shougo

**Comment by Shougo** [comment]
What is the status?

**Comment by Shougo** [comment]
No response.

**Pull request closed without merging**